### PR TITLE
feat: Group chat and Audio player capabilities

### PR DESCRIPTION
**Pull request by hirbod**
# Why

Adds group chat and the audio player + all native libraries needed to run this (currently hidden, will be activated via OTA).
Upload and Composer toolbar will be done in a separate PR.

# How

# Test Plan

### CODE REVIEW

File: apps/expo/app.config.ts [hunk 1]
```diff
@@ -67,6 +67,7 @@ const expoConfig: ExpoConfig = {
   ios: {
     bundleIdentifier: config.scheme,
     buildNumber: majorVersion.toString(),
+    usesIcloudStorage: true,
```
Comment (hirbod) [inline apps/expo/app.config.ts:70]: Needed for expo-documents-picker to show iCloud files

File: packages/app/components/audio-player/audio-player.tsx [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+          id: id,
+          url:
+            url ||
+            "https://showtime-media-stage.b-cdn.net/hirbod-test/AUDIO-2023-07-17-20-38-40.mp3",
+          title: "Showtime",
+          artist: "Creator channels",
+          artwork: "https://media.showtime.xyz/assets/st-logo.png",
+        },
+      ]);
+    }, [id, url]);
```
Comment (hirbod) [inline packages/app/components/audio-player/audio-player.tsx]: This is dummy, will be replaced once the API works correctly

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM 🔥

**Review by alantoa — APPROVED** [review]
LFG!

**Comment by hirbod** [comment]
Superseded by #2442

**Pull request closed without merging**